cylc/uiserver/tests/test_uiserver.py

```diff
@@ -28,32 +28,32 @@ def test_my_application():
 
 def test_cylcuiserver_absolute_path():
     """Test a Cylc UI server created with absolute path for static assets."""
-    cylc_uiserver = CylcUIServer(8000, '/static/path', '/users/test')
+    cylc_uiserver = CylcUIServer(8000, '/users/test', static='/static/path')
     assert cylc_uiserver._port == 8000
-    assert cylc_uiserver._static == '/static/path'
+    assert cylc_uiserver.ui_path == '/static/path'
```

Comment:
Thanks for the fixes, Bruno!

Reply:
Thank **you** for fixing CI for the UIS Ronnie!